R/fd_inventory.R

```diff
@@ -0,0 +1,74 @@
+# Inventory data
+
+
+#' Return the raw inventory table.
+#'
+#' @details The columns are as follows:
+#' - `Site` (character): Replicate code. Each replicate contains
+#'   multiple plots with different disturbance treatments.
+#' - `SubplotID` (integer): Subplot ID number. These subplot codes are a
+#' concatenation of the plot (\code{\link{fd_plots}}) and
+#' subplot \code{\link{fd_subplots}} codes.
+#' - `Tag` (integer): Tree tag number.
+#' - `Species` (character): Species code from the USDA Plants Database; see
+#' \url{https://plants.sc.egov.usda.gov/java/}.
+#' - `DBH_cm` (numeric): Diameter at breast height (1.37m), cm.
+#' - `Health_status` (character): live (L) or dead (D).
+#' - `Canopy_status` (character): Canopy status: overstory dominant (OD),
+#' overstory submissive (OS), XXX (SA), understory (UN).
```

Comment:
Should this be a factor then?

Reply:
I'm torn about this. Those things are true, but factors are very non-intuitive for beginning R users, and in my experience a source of frequent bugs. Kicking this to #11